### DISCUSSION TITLE
RX id's

### DISCUSSION THREAD

**Discussion by Tygrys-1** [pascallanger/DIY-Multiprotocol-TX-Module#536] [Q&A]
Is the RX id (in OpenTx) unique by protocol or global?
I have few flysky receivers and a few frsky ones. Can I use the same RX id's for them, ie. for flysky 1 to 5 and frsky 1 to 5, or should all of them have unique id regardless of the protocol?

**Comment by pascallanger** [comment]
The RX number is per protocols/sub_protocols.
